tests/checks/integration/test_nginx.py

```diff
@@ -22,27 +22,42 @@ def setUp(self):
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['first_one'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://dummyurl:44441/nginx_status/',
                     'tags': ['dummy'],
+                    'nginx_ssl_verify': True,
                 },
                 {
                     'nginx_status_url': 'http://localhost:44441/nginx_status/',
                     'tags': ['second'],
+                    'nginx_ssl_verify': True,
```

Comment:
Yeah, I agree, and it's doubly useless in this test, since True is the default :)
My thought was that if it was at least added in there, it wouldn't be forgotten :P